src/generator.rs

```diff
@@ -117,33 +98,18 @@ pub async fn generate(language: Language) -> eyre::Result<()> {
         out
     };
 
+    generator
+        .run(&out)
```

Comment:
rename `out` to `out_dir` for clarity; avoids the reader having to read the code block to determine what `out` refers to